README.md

```diff
@@ -66,31 +67,44 @@ Before installing deeprankcore you need to install some dependencies. We advise
   - Here, `${TORCH}` and `${CUDA}` should be replaced by the pytorch and CUDA versions installed. You can find these using:
     - `python -c "import torch; print(torch.__version__)"` and
     - `python -c "import torch; print(torch.version.cuda)"`
+      - if this returns `None`, use `cpu` instead
   - For example: `https://data.pyg.org/whl/torch-2.0.0+cpu.html`
-* Only if you have a MacOS with M1 chip, run also:
-  * `conda install pytables`
+* Check if gcc is installed: `gcc --version`.
+  * if this gives an error, run `sudo apt-get install gcc`.
+* For MacOS with M1 chip (otherwise ignore this): `conda install pytables`
 
 ### Deeprank-Core Package
 
 Once the dependencies installed, you can install the latest release of deeprankcore using the PyPi package manager:
 
-```
+```bash
 pip install deeprankcore
 ```
 
 You can get all the new developments by cloning the repo and installing the code with
 
-```
+```bash
 git clone https://github.com/DeepRank/deeprank-core
 cd deeprank-core
 pip install -e ./
 ```
 
-## Documentation
+## Test installation
+
+To check that all components were installed correctly, run either of the following commands and ensure that all tests pass. 
```

Comment:
This is possible if the user has cloned the repo only, otherwise, there will be no tests/ folder. I'd just move this paragraph below the installation using the cloned repo. Also theoretically if the user installs the package from the version deployed on pypi he/she shouldn't care about testing that the package works (because it should); I know I specified the testing part in the tutorials but indeed I was a bit uncertain about that